tests/tests/swfs/avm2/stage_mousechildren/test.toml

```diff
@@ -0,0 +1 @@
+num_ticks = 1
```

Comment:
Small request: can please you add a file-ending newline?
I'll try setting .editorconfig to enforce this in the future.